utils/forms.ts

```diff
@@ -1,34 +1,23 @@
-import CHILD_FORMS from 'data/googleForms/childForms';
-import ADULT_FORMS from 'data/googleForms/adultForms';
+import ADULT_FORM_NAMES from 'data/formFilterNames/adultFormNames';
+import CHILD_FORM_NAMES from 'data/formFilterNames/childFormNames';
 
 import { getPermissionFlag, UserPermissions } from 'utils/user';
 
-type Form = typeof ADULT_FORMS[number];
+const sortForms = (formA: string, formB: string) =>
```

Comment:
As we are dealing with an array of string this sort is superfluous, we can just use `.sort()`